src/server/controllers/ApiController.js

```diff
@@ -48,24 +50,54 @@ export default class ApiController {
       .catch(err => response(res, err.message, 401));
   }
 
+  static deleteEffectsHandler(req, res) {
+    const { title } = req.body;
+    const { userId } = req;
+    User
+      .findOneAndUpdate({ _id: userId, 'effects.title': { $in: title } },
+        { $pull: { effects: { title } } })
+      .exec((err, data) => {
```

Comment:
This exec callback could be moved to another function to not repeat code

Reply:
Done